doc/nrf/ug_zigbee.rst

```diff
@@ -41,12 +41,16 @@ See :ref:`nrfxlib:zboss` for more information.
 Required libraries and drivers
 ******************************
 
-|zboss_lib|
-This library is available in `nrfxlib`_.
+Zigbee requires the following modules to properly operate in |NCS|:
 
-The OSIF subsystem acts as the linking layer between the ZBOSS stack and |NCS|.
-It implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
-The files that handle the OSIF integration are located in :file:`subsys/zigbee/osif`.
+* :ref:`nrfxlib:zboss` available in nrfxlib, with the OSIF subsystem acting as the linking layer between the ZBOSS stack and |NCS|.
+  OSIF implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
+  The files that handle the OSIF integration are located in :file:`subsys/zigbee/osif`.
```

Comment:
```suggestion
  The files that handle the OSIF integration are located in :file:`nrf/subsys/zigbee/osif`.
```
To make it clearer that this is in the nrf repo and not nrfxlib?

Reply:
Good point, I was wondering how to make it more clear, thanks.